pkg/common/unittestcommon/utils.go

```diff
@@ -210,3 +211,12 @@ func (f *fakeVolumeOperationRequestInterface) StoreRequestDetails(
 	f.volumeOperationRequestMap[instance.Name] = instance
 	return nil
 }
+
+// GetNodesForVolumes returns nodeNames to which the given volumeIDs are attached
+func (c *FakeK8SOrchestrator) GetNodesForVolumes(ctx context.Context, volumeID []string) map[string][]string {
+	log := logger.GetLogger(ctx)
+	log.Debugf("GetNodesForVolumes for FakeK8SOrchestrator not yet implemented")
```

Comment:
Is this log needed for unit tests ? Can you confirm if this log is emitted in the stdout ?

Reply:
Not really. I have removed it.